INAD_API_extraction.yaml

```diff
@@ -16,14 +17,24 @@ info:
 
     - [NONBLOCK_PULL_REST] Not Blocking Pull REST
   contact:
+  
+    # da verificare  
```

Comment:
AgID CHECK

Reply:
Aggiornamento:

Pattern Model applicati:
- [ID_AUTH_CHANNEL_01] Direct Trust Transport-Level Security
- [BLOCK_REST] Blocking REST
- [ID_AUTH_REST_02] Direct Trust con certificato X.509
- [INTEGRITY_REST_01] Integrità del payload del messaggio REST
- [NONBLOCK_PULL_REST] Not Blocking Pull REST